src/main/java/com/perficient/movies/model/dtos/ActorDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.perficient.movies.model.dtos;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@Setter
+@Getter
+public class ActorDTO {
```

Comment:
Use ActorDto, search in google about cammel case in java

Reply:
Got it. 👍